src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -91,6 +91,63 @@ def setMessageFields(String messageType, String artifact) {
     return [ 'topic': topic, 'properties': messagePropertiesString, 'content': messageContentString ]
 }
 
+/**
+ * Method that uses contra-lib shared library
+ * to create the ci.artifact.test.messageType messages
+ * @param messageType: queued, running, complete, error
+ * @param artifact: dist-git-pr, koji-build
+ * @return
+ */
+// Plan is to rename to setMessageFields and remove function above once everything seems fine and stable
+def setTestMessageFields(String messageType, String artifact) {
+    // Set values that go in multiple closures
+    myTopic = "${MAIN_TOPIC}.ci.${artifact}.test.${messageType}"
+    myType = (artifact == 'koji-build') ? 'tier0' : 'build'
+    myComponent = env.fed_repo
+    myIssuer = env.fed_owner ?: fed_username
+    // PR builds are scratch so defaulting to scratch = true
+    myScratch = env.isScratch ? env.isScratch.toBoolean() : true
+    taskid = env.fed_task_id ?: env.fed_id
+    myId = (artifact == 'koji-build') ? taskid : env.fed_rev
```

Comment:
@thrix please ack/nack using the fed_rev for the id field in the case of a dist-git-pr (this goes in the header and artifact closures)

Reply:
@johnbieren what is fed_rev please?

Reply:
Either the koji task id or "PR-X" where X is the pr number. We can change that if something else fits better there

Reply:
I do not think that is sufficient the artifact ID should be unique. What about using pull request id instead?

Reply:
Sure thing, I'll push an update soon

Reply:
@thrix the pull request uid? The id is only unique per repo.. unless that is what you meant by unique